templates/kotlin/docs/java/example.md.twig

```diff
@@ -1,71 +1,67 @@
+{% import 'kotlin/base/utils.twig' as utils %}
 import {{ sdk.namespace | caseDot }}.Client
 {% if method.parameters.all | filter((param) => param.type == 'file') | length > 0 %}
 import {{ sdk.namespace | caseDot }}.models.InputFile
 {% endif %}
 import {{ sdk.namespace | caseDot }}.services.{{ service.name | caseUcfirst }}
 
-public void main() {
-    Client client = Client(context)
+Client client = Client(context)
```

Comment:
Why are we removing the main function?

Reply:
Same again, just seems unnecessary and distracting as you can run the code in any function, why specify `main`